about.html

```diff
@@ -0,0 +1,142 @@
+        <p id="second-para">Creatives increasingly understood the importance of collaboration, going beyond the obvious and using new mediums in their work, such as AR, VR, NFTs, etc. The AR Art Summit was born as a reflection of the current shifts happening in the art world.
+        </p>
+        </p>
+      </div>
+      <div class="contact-info">
+        <div class="info-msg">
+          <p>Please contact us per Email for any further questions about AR Art Summit 2022!</p>
+        </div>
+        <div class="email-info">
+          <p>arartsummit2022@arkorea.org</p>
+        </div>
+      </div>
+    </section>
+    <section id="logo-sctn">
+      <header class="logo-headr">
+        <h2>AR Art Summit 2022 Logo</h2>
+        <div class="orange-bar"></div>
+      </header>
+      <div class="logo-info">
+        <p>The logo of AR Art Summit 2022 was decided through the logo competition from 8 June to 7 July.</p>
+      </div>
+      <div id="logo-image">
+        <img src="imgs/ar-art-summit-2022-600x315.jpeg" alt="summit logo">
+      </div>
+      <br>
+    </section>
+    <section class="past-summits">
+      <header class="past-header">
+        <h2>See the Past AR Art Summits</h2>
+        <div class="orange-bar"></div>
+      </header>
+      <div class="past-sub">
+        <p>Take a look at the last two AR Art Summits which took place in Beunos Aires and in Warsaw.</p>
+      </div>
+      <div class="about-years">
+        <div class="about-2013">
+          <img src="imgs/about_2013_hover.png" alt="2 men in 2013 summit">
+        </div>
+        <div class="about-2011">
+          <img src="imgs/about_2011_hover.png" alt="groupie in 2011 summit">
+        </div>
+      </div>
+    </section>
+    <section class="partner">
+      <header class="partner-header">
+        <h2>Partners</h2>
+        <div class="orange-bar"></div>
+      </header>
+      <div class="sponsor-group">
+        <div class="sponsors">
+          <img src="imgs/mozilla.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/googlekorea-removebg-preview.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/naver-removebg-preview.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/kakao-removebg-preview.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/airbnb-removebg-preview.png" alt="">
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer>
+    <div class="footer-logo">
+      <img id="mobile-footer" src="imgs/cc-logo2.png" alt="summit logo">
+      <img src="imgs/logo_cc_w_147x35.png" id="desktop-footer" alt="">
+    </div>
+    <div class="footer-note">
+      <p>2022 Creative Commons Korea.</p>
+      <p>Some Rights Reserved.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] the about page footer looks awesome and perfect, kindly apply the same alignment for the index footer as well

> TIP: in general the content of the footer should be the same on all pages , so you can put the same layout and text of the about page in the footer of index page and the colors can stay the same white for index & dark for about 💯 

About Footer :
![image](https://user-images.githubusercontent.com/89282221/190821656-ca14411e-5992-4b3a-b83f-267ec5dfb944.png)

index footer : 
![image](https://user-images.githubusercontent.com/89282221/190822356-6ef6089d-d5c2-41f3-9f5a-d053eba0e87f.png)